container-compose-dev.yml

```diff
@@ -34,19 +34,27 @@ services:
       - target: 1080
         published: 1080
         protocol: tcp
+        app_protocol: http
         mode: host
+        name: "port for http connections to proxy"
```

Comment:
can this metadata be used by tools or something?

Reply:
yes, https://github.com/compose-spec/compose-spec/blob/master/spec.md#long-syntax-3